src/snowflake/snowpark/session.py

```diff
@@ -21,7 +21,6 @@
 import pkg_resources
 
 from snowflake.connector import ProgrammingError, SnowflakeConnection
-from snowflake.connector.config_manager import _get_default_connection_params
```

Comment:
We probably can do `is_in_stored_procedure()` check for the import? And for safety, also add the same check in the if clause before calling the function.

Reply:
done